components/FormFlashcard/FormInput.js

```diff
@@ -5,15 +5,24 @@ export default function FormInput({
   maxlength,
```

Comment:
Please think about naming camelcase.
```suggestion
  maxLength,
```